src/server/services/duplicatesService.js

```diff
@@ -1,33 +1,33 @@
 const fs = require('fs');
 
 const getDuplicates = require('./helpers/getDuplicates');
+const getFileIndex = require('./helpers/readFileIndex');
 
-const inputFilePath = "src/inputData/";
-const outputFilePath = "src/outputData/";
+const inputFilePath = 'src/inputData/';
+const outputFilePath = 'src/outputData/';
 
-const indexFilePath = "./src/dataBase/file-indexData.json";
+const indexFilePath = './src/dataBase/fileIndexData.json';
 
-module.exports = function() {
-    const getFileIndex = require('./helpers/readFileIndex');
+let fileName =``;
 
+module.exports = function() {
     const indexData = fs.readFileSync(indexFilePath);
     const currentIndex = getFileIndex(indexData);
 
-    const fileName = "duplicates_(" + (currentIndex) + ").json";
+    fileName = `duplicates_(${currentIndex}).json`;
 
     const inputStringFile = fs.readFileSync(inputFilePath + fileName).toString();
     const inputArrFile = inputStringFile.replace(/[{}\n\r"]/igm, '').split(',');
 
-    let cleanedInputArr = inputArrFile.map((val) => val.trim().split(':'));
-    cleanedInputArr = cleanedInputArr.map((val) => [val[0], val[1].trim()]);
-
-    const duplicates = getDuplicates(cleanedInputArr);
-    fs.writeFileSync(outputFilePath + fileName, JSON.stringify(duplicates, null, 2), 'utf-8');
-};
-
-
-
-
+    const cleanedInputArr = inputArrFile.map((val) => val.trim().split(':')).map((val) => [val[0], val[1].trim()]);
```

Comment:
To simplify reading
`inputArrFile`
`    .map((val) => val.trim().split(':'))`
`    .map((val) => [val[0], val[1].trim()]);`